### ISSUE TITLE
Access Denied after using "Empty Wishlist" button

### ISSUE THREAD

**Issue by azumukupoe** [tfedor/AugmentedSteam#578]

**Comment by MxtOUT** [comment]
Could you explain that issue?

**Comment by MattDahEpic** [comment]
It's steam banning the IP for a while after you hit too many 429s. They think it's abuse when you nuke a large wishlist one item at a time. Same goes for the lowest price listings on the market page if the user has 110 products selected.

**Comment by candela97** [comment]
Yeah this is probably due to rate limiting. According to this guide https://steamcommunity.com/sharedfiles/filedetails/?id=1746978201, the author mentions safe a limit of 50 requests. Maybe we should add a warning if one tries to remove more than 50 items at once.

**Comment by candela97** [comment]
There're also a few possible enhancements for this feature, like graceful handling of network errors, adding an "abort" button to abort the operation... but I'm not sure if it's worth it now since clicking the native "remove" link is probably a safer way to do this, and I doubt people are clearing their wishlists often.
